tests/foreman/ui/test_errata.py

```diff
@@ -1120,6 +1120,72 @@ def test_positive_check_errata(session, module_org_with_parameter, registered_co
         assert read_errata['Content']['Errata']['table'][0]['Errata'] == CUSTOM_REPO_ERRATA_ID
 
 
+@pytest.mark.tier3
+@pytest.mark.rhel_ver_match('[8, 9]')
+def test_positive_host_content_library(
+    registered_contenthost,
+    function_lce,
+    module_lce,
+    session,
+):
+    """Check if the applicable errata are available from the content host's Library.
+        View errata table from within All Hosts, and legacy Contenthosts pages.
+
+    :id: a0694930-4bf7-4a97-b275-2be7d5f1b311
+
+    :Setup:
+        1. Multiple environments are present, we will use 'Library'.
+        2. A registered host's Library environment has some additional errata.
+
+    :steps:
+        1. Install the outdated package to registered host, making an errata applicable.
+        2. Go to new All Hosts -> Select the host -> Content -> Errata Tab.
+        3. Go to Legacy Content Hosts -> Select the host -> Errata Tab -> 'Library' env.
+        4. Search for the errata by id. Then, check the entire table without filtering.
+
+    :expectedresults: The expected errata id present in Library is displayed.
+        Only a single errata is present, the tables match between the two pages.
+
+    :parametrized: yes
+    """
+    client = registered_contenthost
+    hostname = client.hostname
+
+    assert client.applicable_errata_count == 0
+    assert client.execute(f'yum install -y {FAKE_1_CUSTOM_PACKAGE}').status == 0
+    assert client.applicable_errata_count == 1
+    assert client.applicable_package_count == 1
+
+    with session:
+        session.location.select(loc_name=DEFAULT_LOC)
+        # check new host > host > content > errata tab:
+        host_tab_search = session.host_new.get_errata_table(
+            entity_name=hostname,
+            search=f'errata_id="{CUSTOM_REPO_ERRATA_ID}"',
+        )
+        # found desired errata_id by search
+        assert len(host_tab_search) == 1
+        assert host_tab_search[0]['Errata'] == CUSTOM_REPO_ERRATA_ID
+        # no filters passed, checking all errata present
+        host_tab_erratum = session.host_new.get_errata_table(hostname)
+        # only the expected errata_id is found
+        assert len(host_tab_erratum) == 1
+        assert host_tab_erratum[0]['Errata'] == CUSTOM_REPO_ERRATA_ID
+        # check legacy chost > chost > errata tab -- search:
+        single_chost_search = session.contenthost.search_errata(
+            hostname, CUSTOM_REPO_ERRATA_ID, environment='Library Synced Content'
+        )
+        # found desired errata_id by search
+        assert len(single_chost_search) == 1
+        assert single_chost_search[0]['Id'] == CUSTOM_REPO_ERRATA_ID
+        # display all entries in chost table, only the expected one is present
+        all_chost_erratum = session.contenthost.search_errata(
+            hostname, errata_id=' ', environment='Library Synced Content'
```

Comment:
do you need the blank `errata_id`?

Reply:
We discussed in slack that this should be changed similar to https://github.com/SatelliteQE/airgun/pull/1152/files#diff-bc6c9d3673e70245978c9c723f3d83335d342eb525f7197b79073d1d0e80a5e2R313 but it can be addressed in a future PR for airgun.